### PR TITLE
docs: document security chain change in 3.10.19 upgrade note

### PR DESCRIPTION
**Pull request by marcambier**
**Issue**

https://github.com/gravitee-io/issues/issues/7824

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/docs-plansbreakingchange-3-10/index.html)

### CODE REVIEW

**Review by Okhelifi — APPROVED** [review]

**Review by ThibaudAV — APPROVED** [review]

**Review by phiz71 — APPROVED** [review]

File: pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Before this version :**
+
+- API key plan is executed if the request contains an API key
+- JWT plan is executed if the request contains a Bearer token
+
+**From this version :**
+
+Security chain has been improved to select and execute plans more efficiently.
+That causes some behavior changes.
```
Comment (ivan-gravitee) [inline pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc:17]:
```suggestion
```

File: pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Api keys and JWT plans are executed only if there is an active subscription related to the provided security token.
```
Comment (ivan-gravitee) [inline pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc:19]:
```suggestion
```

File: pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+If there is no relevant subscription, the plans won't be executed and security chain will continue the parse following plans.
```
Comment (ivan-gravitee) [inline pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc:20]:
```suggestion
```

File: pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Here are some concrete examples of behavior changes :
```
Comment (ivan-gravitee) [inline pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc:22]:
```suggestion
Specific examples of this change are provided in the table below:
```

File: pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+|===
+|API plans | Request |Before this version |From this version
```
Comment (ivan-gravitee) [inline pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc:24]:
```suggestion
| API plans | Request | Prior to this version | From this version
```

File: pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+|API key plan + Keyless plan
+|Request contains an invalid API key
+|Api key plan is executed = HTTP 401 unauthorized
```
Comment (ivan-gravitee) [inline pages/apim/3.x/installation-guide/upgrades/3.10.19/README.adoc:28]:
```suggestion
|API key plan is executed = HTTP 401 unauthorized
```

**Review by ivan-gravitee — CHANGES_REQUESTED** [review]
Some minor suggestions added

**Review by ivan-gravitee — APPROVED** [review]
LGTM

**Pull request merged**